### DISCUSSION TITLE
Is it possible to escape shortcuts of joplin

### DISCUSSION THREAD

**Discussion by LintaoAmons** [ThibaultJanBeyer/joplin-sheets#9] [General]
I want to toggle `goto anything` by Joplin shortcuts after finishing sheet editing. But it can't.

**Comment by ThibaultJanBeyer** [comment]
Could you please write down steps by step what you did?
I'm relatively new to Joplin, what is goto anything? How to use it? What happens? What are your steps?
Thank you!
